shakedown/dcos/command.py

```diff
@@ -158,3 +172,78 @@ def run_dcos_command(command, raise_on_error=False, print_output=True):
             return_code, command, stdout, stderr))
 
     return stdout, stderr, return_code
+
+
+class HostSession:
+    """Context manager that returns an SSH session to run commands.
+    
+    """
+    def __init__(self, host, username, key_path, verbose):
+        self.host = host
+        self.username = username
+        self.key_path = key_path
+        self.verbose = verbose
+        self.exit_code = -1
+        self.output = ''
+        self.session = None
+    
+    def __enter__(self):
+        """Called when using a `with` closure.
```

Comment:
Context Managers aren't closures, are they?

Reply:
I will push another commit that fixes the comment strings.